src/app/movies/movies.component.ts

```diff
@@ -1,43 +1,23 @@
 import { Movie } from '../interfaces';
-import { Component, OnDestroy, OnInit } from '@angular/core';
-import { BaseHttpService } from './../http/base-http.service';
-import { Subscription } from 'rxjs';
+import { Component } from '@angular/core';
 
 @Component({
   selector: 'app-movies',
   templateUrl: './movies.component.html',
   styleUrls: ['./movies.component.css']
 })
-export class MoviesComponent implements OnInit, OnDestroy {
-  movies: Movie[] = [];
-  // movie: Movie = { id: '', type: 'youtube | vimeo' };
-  movie: Movie[] = [];
-  private subscriptions = new Subscription();
 
-  constructor(private baseHttpService: BaseHttpService) { }
+export class MoviesComponent {
+  movies: Movie[] = [];
+  newMovieInput: Movie[] = [];
```

Comment:
jaki sens ma to że input to array filmów? Przeciez input to string, to co user wpisuje w input :)